openupgradelib/openupgrade_130.py

```diff
@@ -0,0 +1,179 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+"""Tools specific for migrating from Odoo 12.0 to 13.0.
+"""
+
+import logging
+from . import openupgrade
+
+_logger = logging.getLogger('OpenUpgrade')
+_logger.setLevel(logging.DEBUG)
```

Comment:
Don't do this. Should be set from configuration. If messages are important enough that by default you want to show them, make them info messages.

Reply:
Alright, I removed the line.